crates/astria-cli/src/types.rs

```diff
@@ -251,6 +255,81 @@ struct SequencerConfig {
     chain_id: String,
 }
 
+#[derive(Debug, PartialEq, Serialize, Deserialize)]
+pub struct CoreResources {
+    conductor: ServiceResources,
+    composer: ServiceResources,
+    geth: ServiceResources,
+}
+
+impl CoreResources {
+    fn default() -> Self {
+        Self {
+            conductor: ServiceResources::default_low(),
+            composer: ServiceResources::default_low(),
+            geth: ServiceResources::default_high(),
+        }
+    }
+}
+
+#[derive(Debug, PartialEq, Serialize, Deserialize)]
+struct ServiceResources {
+    requests: Resource,
+    limits: Resource,
+}
+
+impl ServiceResources {
+    fn default_low() -> Self {
+        Self {
+            requests: Resource::default_low_request(),
+            limits: Resource::default_low_limit(),
+        }
+    }
+
+    fn default_high() -> Self {
+        Self {
+            requests: Resource::default_high_request(),
+            limits: Resource::default_high_limit(),
+        }
+    }
+}
+
+#[derive(Debug, PartialEq, Serialize, Deserialize)]
+struct Resource {
+    cpu: f32,
+    memory: String,
+}
+
+impl Resource {
+    fn default_low_request() -> Self {
+        Self {
+            cpu: 0.01,
```

Comment:
We could maybe use the milliCPU form for these - the note section in [the docs](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-cpu) recommends that to help avoid providing an invalid value in the future.